### ISSUE TITLE
Show all books currently reading in that section of home screen

### ISSUE THREAD

**Issue by placoderm** [bookwyrm-social/bookwyrm#2481]
**Is your feature request related to a problem? Please describe.**
Now, only five books are displayed in the "Currently Reading" section on the home screen.

**Describe the solution you'd like**
I'd like to see them all!!

This is especially important since there is 
1. (currently) No way to search your own books and
2. There doesn't seem to be a way to increase the number of books shown in a list.

This means that when I want to mark a book read, I may have to go through several screens of lists manually.

Of course and alternate solution would be to not read more than 5 books at a time. But that's just not how I roll.

**Additional context**

This is where I'm talking about:

![2022-12-08 15_23_36-](https://user-images.githubusercontent.com/31570884/206417134-ba3136ce-07d6-4dd0-9c25-d699d829a0ed.png)

**Comment by placoderm** [comment]
I'm guessing this is just a simple default setting that could perhaps be increased to some larger number.

**Comment by placoderm** [comment]
> I don't know if you're aware that there is a shelf for the books you're currently reading.

Yep, I do! Thanks. But if I have more than 15 books that I'm reading (please don't shame me 🤣) then I have to look through each page of 15 in the screen you show.



> I think that expanding the small section on the home screen isn't worth it,

Right. But I'm not proposing increasing the size of that area. If you have more than two books you are reading, then it becomes a left to right slide area. At least in your screen shot you don't have any books you are reading. If you look in my screen shot you can see the scroll bar on that area. It scrolls left to right but it only shows max 5 books. 

So even the label doesn't make sense because it's not the books that you are reading, but rather five books you are reading.

Now, one alternative (which still would not be my first choice) would be to have a link to that shelf. Something like this:
![2022-12-08 15_23_36-](https://user-images.githubusercontent.com/31570884/206515112-77d85658-a627-4cde-afa6-7c055ac36ea5.png)

**Comment by marcusyoung** [comment]
The paging on the shelves is there for performance reasons I believe. I think there would be similar concerns about loading a large list of books in the currently reading scrollable section.

**Comment by placoderm** [comment]
> for performance reasons I believe

My guess was it was either for performance, or simply an arbitrary number. I can't imagine, though, that it would affect performance that much. We're talking about tens of books, not thousands. 

Any case, @mouse-reeve would be the one who knows.

**Comment by marcusyoung** [comment]
See #713 r.e. the pagination

**Comment by placoderm** [comment]
Right. They were talking about a list of 1000 or 600 books being served off of a RasberyPI. Surely there is something larger than 15 and smaller that 600 that is still performant on appropriate hardware, lol.

**Comment by marcusyoung** [comment]
That is something that perhaps could be admin controlled on a per instance basis. There are ongoing performance issues with the main instance which won't be running off a Raspberry Pi!  As the instances are largely self funded by the maintainers there may be a desire to minimise server costs. Anything that reduces overhead will probably be welcome. But flexibility for those that want to up that limit to increase usability would be fine by me. There is also the possibility of auto loading as you scroll down which seems to be the way things have gone on many sites recently.

**Comment by franbarra** [comment]
> > I don't know if you're aware that there is a shelf for the books you're currently reading.
> 
> Yep, I do! Thanks. But if I have more than 15 books that I'm reading (please don't shame me rofl) then I have to look through each page of 15 in the screen you show.
> 

No judgments on the amount of books read (I also have like 9 or 10 books I'm currently reading), but on physical copies.

> > I think that expanding the small section on the home screen isn't worth it,
> 
> Right. But I'm not proposing increasing the size of that area. If you have more than two books you are reading, then it becomes a left to right slide area. At least in your screen shot you don't have any books you are reading. If you look in my screen shot you can see the scroll bar on that area. It scrolls left to right but it only shows max 5 books.
> 

Oh, I misunderstood your original proposition then, my bad.

> So even the label doesn't make sense because it's not the books that you are reading, but rather five books you are reading.
> 
> Now, one alternative (which still would not be my first choice) would be to have a link to that shelf. Something like this: ![2022-12-08 15_23_36-](https://user-images.githubusercontent.com/31570884/206515112-77d85658-a627-4cde-afa6-7c055ac36ea5.png)

It is true that the main instance has been suffering from performance issues so pagination has to be small, but I agree that maybe making the home menu link to the "Currently Reading", "Read" and "To Read" would be a nice improvement. Make the tag link to the corresponding shelf.

The reason I don't agree with the scrollbar having larger displacement (say, 15 or 20 items) is that, since the home menu is pretty small, the scrollbar loses preciseness when trying to scroll horizontally (at least for me), it just goes all whack when trying to move it to a particular book.

I think that making both the tags of the home menu linked with the shelfs and having the possibility of personalizing the pagination on a per-instance basis would be the way to go.
